barretenberg/cpp/src/barretenberg/vm/avm/tests/helpers.test.hpp

```diff
@@ -25,7 +25,7 @@ using FF = bb::AvmFlavorSettings::FF;
 using Row = bb::AvmFullRow<bb::fr>;
 using ThreeOpParam = std::array<FF, 3>;
 using ThreeOpParamRow = std::tuple<ThreeOpParam, bb::avm_trace::AvmMemoryTag>;
-using VmPublicInputsNT = bb::avm_trace::VmPublicInputs<FF>;
+using VmPublicInputsNT = bb::avm_trace::VmPublicInputs_<FF>;
```

Comment:
I think we could rename this one to VmPublicInputs or alternatively we are explicit everywhere by defining VmPublicInputsNT and VmPublicInputsCT (which personally I would prefer).

Reply:
Maybe we could say that the default is native and for the circuit type we are explicit in the name:
`using VmPublicInputs = bb::avm_trace::VmPublicInputs_<FF>;
using VmPublicInputsCT = bb::avm_trace::VmPublicInputs_<field_t<.....>>;
`

Reply:
ah yeah this isnt too bad in the future once we decide which naming scheme we want to use. I just did a mass rename so i didnt run into painful merge conflicts 🫠